python/mleap/bundle/serialize.py

```diff
@@ -16,6 +16,13 @@ def __init__(self, values):
         self.values = values
 
 
+class EncoderNumpySupport(json.JSONEncoder):
```

Comment:
@trams I see there is another PR that contains these changes, could you please remove them from this one?

Reply:
Yes. I divided this into two pull requests